test/e2e/components/vl-alert.js

```diff
@@ -41,7 +41,10 @@ class VlAlert extends VlElement {
     }
 
     async getActions() {
-        return this._innerText('#actions');
+      const slot = await (await this._getAlert()).findElement(
+          By.css("#actions-slot"));
```

Comment:
misschien just me maar zo van die await in await toestanden pikken een beetje aan mijn ogen ...